vault.go

```diff
@@ -0,0 +1,234 @@
+package main
```

Comment:
Oh, ok. You should just be able to move this into `vault` subdirectory and change this line to `package vault`. Let me know if something isn't working right beyond that...